src/utils/media-query-props.js

```diff
@@ -135,13 +171,23 @@ function resolveMediaQueryListExpr(expr) {
 
 /**
  * @param {!ExprDef} expr
- * @param {?function()} onChange
+ * @param {?function()} callback
  */
-function setOnChange(expr, onChange) {
+function setOnChange(expr, callback) {
   for (let i = 0; i < expr.length; i++) {
     const {query} = expr[i];
     if (query) {
-      query.onchange = onChange;
+      // The `onchange` API is preferred, but the IE only supports
+      // the `addListener/removeListener` APIs.
```

Comment:
Nit: If we need `addListener`/`removeListener`, can we just remove the `onchange` use and save the bytes?

Reply:
They are deprecated everywhere else 😂